Dockerfile

```diff
@@ -15,6 +15,13 @@ ENV HOME=/opt/pelias
 WORKDIR ${WORK}
 COPY . ${WORK}
 
+# install node 8.x
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
+    apt-get install -y nodejs
```

Comment:
We'll be merging https://github.com/pelias/dockerfiles/pull/39 fairly soon, so track that issue as this line will then no longer be required :)

Reply:
As it turns out, I already ended up doing this in January on our fork's master :confused: Removing from this PR.

Reply:
Once you merge, I'll update our master. Since I forgot I had done this on another branch (which was merged to our fork master recently), I can report that everything has worked great.